bindings/pyroot/cppyy/CPyCppyy/src/Pythonize.cxx

```diff
@@ -841,6 +841,16 @@ PyObject* name##StringCompare(PyObject* self, PyObject* obj)                 \
 CPPYY_IMPL_STRING_PYTHONIZATION_CMP(std::string, Stl)
 CPPYY_IMPL_STRING_PYTHONIZATION_CMP(std::wstring, StlW)
 
+Py_hash_t StlStringHash(PyObject* self)
```

Comment:
Why not adding this as a pythonization in Python via the PyROOT layer? Is this also there in upstream?

Reply:
> Why not adding this as a pythonization in Python via the PyROOT layer? Is this also there in upstream?

Answered by the PR description, already seen by @stwunsch !